util/otbn_build.py

```diff
@@ -201,7 +201,8 @@ def main() -> int:
         out_embedded_obj = out_dir / (app_name + '.rv32embed.o')
         args = (['-O', 'elf32-littleriscv',
                  '--prefix-sections=.rodata.otbn',
-                 '--set-section-flags=*=alloc,load,readonly',
+                 '--set-section-flags=*=alloc,readonly',
+                 '--set-section-flags=.data=load',
```

Comment:
I don't think I understand this. I would have thought you want every section to be ALLOC, LOAD, READONLY except for the new BSS section, which should be ALLOC, READONLY.

If I've understood right, this sets `.text` to ALLOC, READONLY (what happened to LOAD?!) and `.data` to LOAD (it should also be READONLY).

Doesn't something like this work?
```
                 '--set-section-flags=*=alloc,load,readonly',
                 '--set-section-flags=.bss=alloc,readonly',
```

Reply:
Yes, it does! I misunderstood the behavior of `set-section-flags`, and thought it would set everything to `alloc, readonly` and set `.data` additionally to `load` (but it's true `.text` should be `load` also). Your suggestion works locally; I'll update the PR to match.